docs/stencil-docs/reference-docs/front-matter-reference.md

```diff
@@ -2,32 +2,31 @@
 
 <div class="otp" id="no-index">
 
-### On This Page
-- [Supported Templates](#supported-templates)
-- [Global Attributes](#global-attributes)
-- [Category Attributes](#category-attributes)
-- [Blog Attributes](#blog-attributes)
-- [Product Attributes](#product-attributes)
-- [Brand Attributes](#brand-attributes)
-- [Brand List Attributes](#brand-list-attributes)
-- [Cart Attributes](#cart-attributes)
-- [Search Attributes](#search-attributes)
+### On this page
+- [Supported templates](#supported-templates)
+- [Global attributes](#global-attributes)
+- [Category attributes](#category-attributes)
+- [Blog attributes](#blog-attributes)
+- [Product attributes](#product-attributes)
+- [Brand attributes](#brand-attributes)
+- [Brand list attributes](#brand-list-attributes)
+- [Search attributes](#search-attributes)
   
 </div>
 
 Front matter defines which store resources are available to be rendered within a Stencil template. Front matter is declared at the top of each template and uses [YAML](https://yaml.org/) syntax. For more information, see [Declaring Front Matter Objects](https://developer.bigcommerce.com/stencil-docs/storefront-customization/using-front-matter#declaring-front-matter-objects).
 
-## Supported Templates
-YAML Front Matter is supported for templates in the `templates/pages/` directory. Injecting objects in the front matter of `templates/pages/page.html` will make the objects available to custom templates.
+## Supported templates
+You can use YAML Front Matter for templates in the `templates/pages/` directory. Injecting objects in the front matter of `templates/pages/page.html` will make the objects available to custom templates.
 
 
-Front Matter is not supported for templates in the following directories:
+You cannot use Front Matter for templates in the following directories:
 * `templates/components/`
 * `templates/layout/`
 * `templates/pages/custom/`

```

Comment:
I believe this is now supported actually (custom pages). @junedkazi can you confirm it is supported on storefront service?